src/gba/audio.c

```diff
@@ -224,6 +227,16 @@ void GBAAudioWriteWaveRAM(struct GBAAudio* audio, int address, uint32_t value) {
 	audio->psg.ch3.wavedata32[address | (!audio->psg.ch3.bank * 4)] = value;
 }
 
+uint32_t GBAAudioReadWaveRAM(struct GBAAudio* audio, int address) {
+	int bank = !audio->psg.ch3.bank;
+
+	// When the audio hardware is turned off, bank 0 is selected.
+	if (!audio->enable)
+		bank = 1;
```

Comment:
```suggestion
	if (!audio->enable) {
		bank = 1;
	}
```

Reply:
Further, it the comment can be a bit confusing given the fact that it says "0" but assigns 1. If you put the negation after the block it'll probably be clearer.

Reply:
I see what you mean. I've reworded the comment a bit, what do you think now?

Also, I forgot to commit the same check in the function to write to wave RAM, I've added the change to the commit now.